src/classes/STG_InstallScript.cls

```diff
@@ -338,24 +343,37 @@ global without sharing class STG_InstallScript implements InstallHandler {
         }
     }
     
-    /**********************************************************************************************************
-    * @description Enables prevent deletion by default on Hierarchy Custom Settings for new installs and 
-    * when upgrading to 1.95
+    /**************************************************************************************************************************
+    * @description Enables prevent deletion by default on Hierarchy Custom Settings for new installs and when upgrading to 1.95
```

Comment:
One small thing, I know this has been merged already but could we make the description a little bit specific. After all, they are all enabling deletion prevention?

Reply:
good catch, updated @xinnanlisf